src/app/api/staking/getDeposit.ts

```diff
@@ -0,0 +1,33 @@
+import Tenant from "@/lib/tenant/tenant";
+import prisma from "@/app/lib/prisma";
+import { cache } from "react";
+import { StakedDeposit } from "@/lib/types";
```

Comment:
I've been colocating types for a feature within the feature route itself -- do we want to be putting all of our types in one single type file?